src/parser/shaman/elemental/modules/talents/PrimalFireElemental.js

```diff
@@ -9,16 +9,16 @@ import Analyzer from 'parser/core/Analyzer';
 
 import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
 
-const damagingCasts = [SPELLS.METEOR.id, SPELLS.IMMOLATE.id, SPELLS.FIRE_ELEMENTAL_FIRE_BLAST.id];
+const damagingCasts = [SPELLS.FIRE_ELEMENTAL_METEOR.id, SPELLS.FIRE_ELEMENTAL_IMMOLATE.id, SPELLS.FIRE_ELEMENTAL_FIRE_BLAST.id];
 
 class PrimalFireElemental extends Analyzer {
   meteorCasts = 0;
   PFEcasts = 0;
 
   usedCasts = {
-    meteor: false,
-    immolate: false,
-    fire_blast: false,
+    'Meteor': false,
+    'Immolate': false,
+    'Fire Blast': false,
```

Comment:
did you consider using the spell ids as indices? like 

```js
usedCasts = {
    [SPELS.FIRE_ELEMENTAL_METEOR.id]: false,
    ...
}
```

this could simplify your switch later

Reply:
It would make the generation of the string harder/awkward tho. You got a good idea to make that work when using the spellids?

Reply:
oh i see that now. you could do `unusedSpells.map(spellId => this.abilities.getAbility(spellId).name).join(', ')`. not quite as clean, but functional.

imho its worth the tradeoff, but i'll leave it up to you. if you don't think its worth just fix the spaces and i'll merge.